dandi/cli/cmd_organize.py

```diff
@@ -34,6 +34,22 @@
     default="auto",
     show_default=True,
 )
+@click.option(
+    "--modify-external-file-fields",
+    is_flag=True,
+    default=False,
+    help="using this options will rewrite the 'external_file' argument of the "
+    "ImageSeries in the nwb file. The value written will correspond to the"
+    " new location of the video files after being organized. This option will "
+    "work only if --files-mode is copy/move since the original nwb file is "
+    "modified",
+)
+@click.option(
+    "--media-files-mode",
+    type=click.Choice(["copy", "move", "symlink", "hardlink"]),
+    default=None,
+    help="Supply one of copy/move/symlink/hardlink as the value",
```

Comment:
This help string says nothing that isn't already conveyed by the option metavar.  Either make it informative or delete it.

Reply:
thanks, changes here: https://github.com/dandi/dandi-cli/pull/841/commits/4241edca5a9a434ab9d75a071b60c89af8f34bb5